app/views/directives/osc-routing.html

```diff
@@ -170,29 +168,27 @@
     <div ng-show="secureRoute">
       <!-- TLS Termination -->
       <div class="form-group">
-        <label>TLS Termination</label>
-        <select ng-model="route.tls.termination" class="form-control">
-          <option value="edge">Edge</option>
-          <option value="passthrough">Passthrough</option>
-          <option value="reencrypt">Re-encrypt</option>
-        </select>
+        <label for="tlsTermination">TLS Termination</label>
+        <ui-select ng-model="route.tls.termination" input-id="tlsTermination" search-enabled="false">
+          <ui-select-match>{{$select.selected | startCase}}</ui-select-match>
```

Comment:
This will give a different value than before for `reencrypt`. 

We should be able  create an array of object with `id` and `label` attributes and then bind the value as the `id`. That way we can have future values that won't work for `startCase` filter and will let us translate the labels to different languages eventually. See

https://github.com/angular-ui/ui-select/wiki/ui-select-choices#example-binding-single-property